test/OptionSettlementEngine.t.sol

```diff
@@ -653,56 +670,54 @@ contract OptionSettlementTest is Test, NFTreceiver {
     //                            PROTOCOL ADMIN
     // **********************************************************************
 
-    function testSetFeeSwitch() public {
+    function testprotocolFees() public {
         // precondition check -- in test suite, fee switch is enabled by default
-        assertTrue(engine.feeSwitch());
+        assertTrue(engine.feesEnabled());
 
         // disable
         vm.startPrank(FEE_TO);
-        engine.setFeeSwitch(false);
+        engine.protocolFees(false);
 
-        assertFalse(engine.feeSwitch());
+        assertFalse(engine.feesEnabled());
 
         // enable
-        engine.setFeeSwitch(true);
+        engine.protocolFees(true);
 
-        assertTrue(engine.feeSwitch());
+        assertTrue(engine.feesEnabled());
     }
 
-    function testEventSetFeeSwitch() public {
+    function testEventprotocolFees() public {
```

Comment:
Upper-case Protocol — you need to set your find+replace regex to not be so greedy, hehe same thing happened to me